catalog/video/video-mpegts-stream/src/test/java/org/codice/alliance/video/stream/mpegts/rollover/CatalogRolloverActionTest.java

```diff
@@ -114,6 +117,11 @@ public void setup() throws SourceUnavailableException, IngestException {
     when(udpStreamProcessor.getStreamId()).thenReturn(streamId);
     when(udpStreamProcessor.getSubject()).thenReturn(new SimpleSubject());
 
+    packetBuffer = mock(PacketBuffer.class);
+    when(packetBuffer.getLastSegmentStart()).thenReturn(-1l);
```

Comment:
We should have a test for the happy path where both times are positive.

Reply:
Added test